utils/doclint/generateDotnetApi.js

```diff
@@ -343,7 +343,7 @@ function generateNameDefault(member, name, t, parent) {
           attemptedName = attemptedName.substring(0, attemptedName.length - 1);
         let probableType = additionalTypes.get(attemptedName);
         if ((probableType && typesDiffer(t, probableType))
-          || (["Value"].includes(attemptedName))) {
+          || (["Value", "File"].includes(attemptedName))) {
```

Comment:
In java we ended up naming this type `FilePayload` (special cased in the java generator too), perhaps we could use the same name in C#

Reply:
I have that name actually. I bet that's the name in Typescript.
@avodovnik can we declare that in the md file or we need to add some code here?

Reply:
Do you buy a Map like this @avodovnik https://github.com/microsoft/playwright-java/blob/7ca9b33f46422a7333bae22d74eacd5a0d209967/tools/api-generator/src/main/java/com/microsoft/playwright/tools/ApiGenerator.java#L256 ?